### DISCUSSION TITLE
git lfs migrate import --no-rewrite <filepath> doesn't catch all files repeatedly, and warns that some files are not being tracked that are

### DISCUSSION THREAD

**Discussion by ghost** [git-lfs/git-lfs#5157] [Q&A] [answered]
Two things, postive and negative mistakes (I don't know which is which but they seem inverse):
I am using `git lfs migrate import --no-rewrite <filepath>` to make `git lfs track` work because as of now it's broken on its own.
However, it doesn't catch all files that it should with the correct file path (using * or specific filename), so I can't use github for LFS like I am paying for. I've gone through cycles of discontinuing my payments but it's just so tempting.

Google is a mess. Would not recommend it. A concise answer here would be great.

**Comment by ghost** [comment, marked as answer]
Seems using * symbol to find files and doing it successfully (file was listed as targeted) still did not work like selecting the specific files in question one by one. Or maybe I was just using * incorrectly. Hopefully the latter.

**Comment by bk2204** [comment]
`git lfs migrate import --no-rewrite` precedes the native Git support for this, which is `git add --renormalize .` followed immediately by a commit.  That's what I would recommend in this case provided you have a new enough Git.
